CMakeLists.txt

```diff
@@ -52,30 +52,45 @@ message(STATUS "ROCMLIR_BIN_DIR: ${ROCMLIR_BIN_DIR}")
 set(CMAKE_BUILD_RPATH "${ROCMLIR_LIB_DIR};${LLVM_EXTERNAL_LIB_DIR}")
 message(STATUS "CMAKE_BUILD_RPATH: ${CMAKE_BUILD_RPATH}")
 
+set(CMAKE_INSTALL_PREFIX "/opt/rocm" CACHE STRING "")
```

Comment:
`CMAKE_INSTALL_PREFIX` is only needed when `BUILD_FAT_LIBROCKCOMPILER` is ON. Is it ok to define it on both cases?

Reply:
Probably fine